ip-clearance-arrow-datafusion-comet.xml

```diff
@@ -0,0 +1,261 @@
+                    <title>Copyright</title>
+                    <table>
+                        <tr>
+                            <th>date</th>
+                            <th>item</th>
+                        </tr>
+                        <tr>
+                            <td>2024-02-03</td>
+                            <td>Check and make sure that the papers that transfer rights to the ASF been received. It is
+                                only necessary to transfer rights for the package, the core code, and any new code
+                                produced by the project.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>2024-01-07</td>
+                            <td>The changes in the <a href="https://github.com/apache/arrow-datafusion-comet/pull/1">donation PR</a>
+                                include adding the ASF copyright, verified manually as well as by the automated
+                                RAT check (<a href="https://github.com/apache/arrow-datafusion-comet/pull/1#issuecomment-1913367405">link</a>).
+                            </td>
+                        </tr>
+                    </table>
+                    <p>
+                        Identify name recorded for software grant:
+                        <em>the name of the grant as recorded
+                            in the foundation/officers area, in either grants.txt or cclas.txt, so that the
+                            grant can be easily identified. If recorded in the grants.txt document, use the
+                            "for" or title. If recorded in the cclas.txt document, use the company name
+                            (field 2 without submitter name) and the "form on file" name (field 4, without
+                            any people's names).
+                        </em>
+                    </p>
+                </section>
+                <section id="Verify+distribution+rights">
+                    <title>Verify distribution rights</title>
+                    <p>
+                        Corporations and individuals holding existing distribution rights:
+                    </p>
+                    <ul>
+                        <li>
+                            <em>Apple</em>
+                        </li>
+                    </ul>
+                    <table>
+                        <tr>
+                            <th>date</th>
+                            <th>item</th>
+                        </tr>
+                        <tr>
+                            <td>2024-02-05</td>
+                            <td>Check that all active committers have a signed CLA on
+                                record.
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>2024-02-03</td>
+                            <td>Remind active committers that they are responsible for
```

Comment:
Has anyone officially reminded the committers they are responsible for ensuring they are authorized their contributions?